Neos.Flow/Classes/Mvc/ActionResponse.php

```diff
@@ -55,6 +56,11 @@ final class ActionResponse
      */
     protected $cookies = [];
 
+    /**
+     * @var ResponseInterface
+     */
+    protected $httpResponse;
```

Comment:
I just realized that all these fields should be `private` (since it's a final class that won't make a difference from the outside anyways)..
But we can "fix" that separately!

Reply:
..and when we do we should mark all remaining fields optional